common/setups/rasr/hybrid_system.py

```diff
@@ -228,10 +228,10 @@ def returnn_training(
         cv_corpus_key,
         devtrain_corpus_key=None,
     ) -> returnn.ReturnnTrainingJob:
-        if nn_train_args.returnn_root is None:
-            nn_train_args.returnn_root = self.returnn_root
-        if nn_train_args.returnn_python_exe is None:
-            nn_train_args.returnn_python_exe = self.returnn_python_exe
+        # if 'returnn_root' not in nn_train_args:
```

Comment:
Careful, this is not user code, but the common code. You must not change this. 
You probably commented this out, because it caused some crash in your setup. As long as the main branch here is not broken, I assume its some wrong configuration? 
If you think its not please make a separate PR with only this change for open review.